services/comps/virtio/src/transport/mod.rs

```diff
@@ -25,6 +25,8 @@ pub mod pci;
 pub trait VirtioTransport: Sync + Send + Debug {
     // ====================Device related APIs=======================
 
+    fn get_device_name(&self) -> String;
```

Comment:
I think we should use `device_id` instead of `get_device_name`, the returned id can be a `u64`.

Reply:
Fixed, we remove the `get_device_name` function from VirtioTransport. Instead, we add the `get_id`  by adding the support of VIRTIO_BLOCK_GET_ID command into the virtio-blk driver.